### ISSUE TITLE
Double implicit not detected

### ISSUE THREAD

**Issue by domialex** [domialex/Sidekick#518]
```
Rarity: Rare
Agony Finger
Amethyst Ring
--------
Requirements:
Level: 59
--------
Item Level: 77
--------
Cannot be Ignited (implicit)
Anger has 20% increased Aura Effect (implicit)
--------
+49 to Dexterity
+78 to Evasion Rating
+35 to maximum Mana
+31% to Lightning Resistance
+7% to Chaos Resistance
0.39% of Physical Attack Damage Leeched as Life
--------
Corrupted

```
![image](https://user-images.githubusercontent.com/4694217/106086708-b29d4180-60f0-11eb-9ae9-84705bc0a7c0.png)

**Comment by domialex** [comment]
It's probably specific to this attribute.

**Comment by leMicin** [comment]
Fixed in 2.0
